### ISSUE TITLE
DisplayLink monitor not working in GNOME Xorg

### ISSUE THREAD

**Issue by evprkr** [AdnanHodzic/displaylink-debian#688]
Basically a fresh install, just did it a few hours ago. I had installed the drivers from DisplayLink themselves, which worked on Wayland, but I had to switch to Xorg because Wayland wasn't working with Steam Link. Now, I'm just getting no signal on my second monitor at all.

Oddly enough, if I log out, I get a signal on the second monitor on the login screen.

Edit: Also, I did follow the post-installation guide and tried all of the troubleshooting steps including uninstalling and reinstalling.

```
--------------- Linux system info ----------------

Distro: Ubuntu
Release: impish
Kernel: 5.13.0-30-generic

---------------- DisplayLink info ----------------

Driver version: 1.9.1
DisplayLink service status: up and running
EVDI service version: 1.9.1

------------------ Graphics card -----------------

Vendor: amdgpu
Subsystem: Inc.
VGA: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] (rev e7)
VGA (3D): 
X11 version: 1.20.13-1ubuntu1.1
X11 configs: /etc/X11/xorg.conf.d/20-displaylink.conf

-------------- DisplayLink xorg.conf -------------

File: /etc/X11/xorg.conf.d/20-displaylink.conf
Contents:
 Section "Device"
    Identifier "AMDGPU"
    Driver     "amdgpu"
    Option     "PageFlip" "false"
EndSection

-------------------- Monitors --------------------

Providers: number : 1
Provider 0: id: 0x5a cap: 0x9, Source Output, Sink Offload crtcs: 6 outputs: 5 associated providers: 0 name:Radeon RX 580 Series @ pci:0000:01:00.0

-------------------------------------------------------------------

```

**Comment by evprkr** [comment]
Output from `dmesg | grep Display` while unplugging and plugging in the DisplayLink adapter:

```
evan@blackbox:~/displaylink-debian$ sudo dmesg | grep Display
[    7.203875] [drm] Display Core initialized with v3.2.132!
[    7.784050] usb 3-2.4.4: Manufacturer: DisplayLink
[    9.313684] evdi: [I] (card0) Opened by Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[    9.314256] evdi: [I] (card0) Connected with Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[  252.045092] evdi: [I] (card0) Disconnected from Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[  252.045201] evdi: [I] (card0) Closed by Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)

evan@blackbox:~/displaylink-debian$ sudo dmesg | grep Display
[    7.203875] [drm] Display Core initialized with v3.2.132!
[    7.784050] usb 3-2.4.4: Manufacturer: DisplayLink
[    9.313684] evdi: [I] (card0) Opened by Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[    9.314256] evdi: [I] (card0) Connected with Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[  252.045092] evdi: [I] (card0) Disconnected from Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[  252.045201] evdi: [I] (card0) Closed by Task 1643 (DesktopManagerE) of process 1617 (DisplayLinkMana)
[  269.113571] usb 3-2.4.4: Manufacturer: DisplayLink
[  269.642011] evdi: [I] (card0) Opened by Task 5576 (DesktopManagerE) of process 5568 (DisplayLinkMana)
[  269.642620] evdi: [I] (card0) Connected with Task 5576 (DesktopManagerE) of process 5568 (DisplayLinkMana)
```

**Comment by evprkr** [comment]
I decided to check that secure boot was disabled _just in case_ even though I had seen during the installation that it was disabled, as I thought it had been, but it was enabled. So I disabled that, rebooted, uninstalled, reinstalled, deleted the conf file, rebooted, and NOW it's working.

However, there is a significant amount of lag/screen tearing -- Not sure if it's relevant, but it's a portrait monitor, flipped in Ubuntu's display settings.
